Automattic/vip-go-mu-plugins
# Update/jetpack9.0.2.1

## Description

Bumps Jetpack in 9.0 folder to 9.0.2.1 version.

There was some issue with subtree complaining about diverged histories. So I end up removing the folder and creating new subtree.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Apply the branch
1. See in jetpack section the version
1. Test all the stuff.
